solidago/src/solidago/pipeline/inputs.py

```diff
@@ -53,17 +53,17 @@ def ratings_properties(self) -> pd.DataFrame:
     def get_individual_scores(
         self,
         user_id: Optional[int] = None,
-        entity_id: Optional[str] = None,
         criteria: Optional[str] = None,
     ) -> pd.DataFrame:
-        raise NotImplementedError
+        """Fetch data about previously computed individual scores
 
-    @abstractmethod
-    def get_collective_scores(
-        self,
-        entity_id: Optional[str] = None,
-        criteria: Optional[str] = None,
-    ) -> pd.DataFrame:
+        Returns:
+        - DataFrame with columns
+            * `user_id`: int
+            * `entity_id`: int
+            * `criteria`: str
+            * `score`: float
+        """
         raise NotImplementedError
```

Comment:
I removed `get_collective_scores()` signature from the abstract `TournesolInput` as we don't plan to use them in pipeline (contrary to the `individual_scores` which will be useful for optimization purposes). So they should not be necessary to implement `TournesolInput` at this point.